src/herder/test/UpgradesTests.cpp

```diff
@@ -841,9 +841,12 @@ TEST_CASE("config upgrades applied to ledger", "[soroban][upgrades]")
             auto& configEntry = configUpgradeSetXdr.updatedEntry.emplace_back();
             configEntry.configSettingID(CONFIG_SETTING_CONTRACT_COMPUTE_V0);
             configEntry.contractCompute().feeRatePerInstructionsIncrement = 111;
-            configEntry.contractCompute().ledgerMaxInstructions = 222;
-            configEntry.contractCompute().txMemoryLimit = 333;
-            configEntry.contractCompute().txMaxInstructions = 444;
+            configEntry.contractCompute().ledgerMaxInstructions =
```

Comment:
We also need a test where we set some values below min and make sure that upgrade is invalid.